### PR TITLE
Fixed agenda items sorting and deleting

### PR DESCRIPTION
**Pull request by BartoszJarocki**
Related issue: https://github.com/ParabolInc/parabol/issues/5056 

Agenda items related relay store updaters were broken because they were reading non-exsiting `agendaItemId` property from `AgendaItemsStage`. 
Screenshot 2021-06-24 at 18 11 29
 
Now, both sorting and deletion work without any issues (currently, on prod sorting and deletion are broken).

### CODE REVIEW

File: packages/server/graphql/mutations/removeAgendaItem.ts [hunk 1]
```diff
@@ -44,10 +44,8 @@ export default {
       return standardError(new Error('Agenda item not found'), {userId: viewerId})
     }
     const filterFn = (stage: AgendaItemsStage) => stage.agendaItemId === agendaItemId
-    const meetingIds = await removeStagesFromMeetings(filterFn, teamId, dataLoader)
-    // safe to do so because we guarantee only 1 action meeting at the same time
-    const [meetingId] = meetingIds
-    const data = {agendaItem, meetingId}
+    await removeStagesFromMeetings(filterFn, teamId, dataLoader)
+    const data = {agendaItem, meetingId: agendaItem.meetingId}
```
Comment (BartoszJarocki) [inline packages/server/graphql/mutations/removeAgendaItem.ts:48]: `const [meetingId] = meetingIds` this was causing to return wrong meeting id when there's more than one meeting active. I've changed it to return `meetingId` strictly related to `agendaItem`

File: packages/client/mutations/UpdateAgendaItemMutation.ts [hunk 2]
```diff
@@ -44,9 +44,8 @@ const handleUpdateAgendaPhase = (
       phase ? phase.getValue('phaseType') === 'agendaitems' : false
     )
     if (!agendaPhase) return
-    const getSortOrder = (stage) => {
-      const agendaItemId = stage.getValue('agendaItemId')
-      const agendaItem = store.get<AgendaItem_agendaItem>(agendaItemId)
+    const getSortOrder = (stage: RecordProxy) => {
+      const agendaItem = stage.getLinkedRecord<AgendaItem_agendaItem>('agendaItem')
```
Comment (mattkrick) [inline packages/client/mutations/UpdateAgendaItemMutation.ts:48]: :facepalm: great find!

File: packages/server/graphql/mutations/removeAgendaItem.ts (hunk 1 above)
Comment (mattkrick) [reply packages/server/graphql/mutations/removeAgendaItem.ts:48]: this makes a lot more sense :+1:

**Review by mattkrick — APPROVED** [review]
fantastic work!

**Pull request merged**